client/modules/IDE/components/Sidebar.jsx

```diff
@@ -41,6 +41,7 @@ class Sidebar extends React.Component {
   }
 
   render() {
+    let isFocused = false;
```

Comment:
The same thing that I said about `FileNode applies here.`

Reply:
I made the required changes.